js/image/shared/Webcam.svelte

```diff
@@ -131,7 +142,20 @@
 		recording = !recording;
 	}
 
-	access_webcam();
+	let webcam_accessed = false;
+
+	function record_video_or_photo(): void {
+		if (mode === "image" && streaming) {
+			recording = !recording;
+		}
+		const func = mode === "image" ? take_picture : take_recording;
+		func();
```

Comment:
let's avoid abbreviations like `func`. Personally think this is more readable tbh
```suggestion
		if (mode === "image") {
		  take_picture();
		} else {
		  take_recording();
		}
```